### PR TITLE
Clean up XML documentation comments (continued)

### PR DESCRIPTION
**Pull request by stakx**
This is the first of approx. 3 additional pull requests regarding issue #1097. I decided to submit one PR per topic area. It should be far easier to review the changes if you can focus on one issue at a time.

The idea behind this PR is to get all the simple stuff & obviously incorrect things out of the way. See the commit message for more details.

### CODE REVIEW

File: Src/Newtonsoft.Json/Utilities/LinqBridge.cs [hunk 1]
```diff
@@ -724,7 +724,7 @@ private static IEnumerable<TSource> ReverseYield<TSource>(IEnumerable<TSource> s
     }
 
     /// <summary>
-    /// Returns an <see cref="Int64"/> that represents the total number 
+    /// Returns a <see cref="Int64"/> that represents the total number
```
Comment (stakx) [inline Src/Newtonsoft.Json/Utilities/LinqBridge.cs]: This is somewhat counterintuitive. At least in Visual Studio's C# code editor, `Int64` will be replaced with the C# keyword `long` in IntelliSense tooltips. "A long…" reads better than "An long…".

File: Src/Newtonsoft.Json/Linq/JToken.cs [hunk 2]
```diff
@@ -1915,7 +1915,7 @@ public static JToken FromObject(object o, JsonSerializer jsonSerializer)
         }
 
         /// <summary>
-        /// Creates the specified .NET type from the <see cref="JToken"/>.
+        /// Creates an object of the specified .NET type from the <see cref="JToken"/>.
```
Comment (JamesNK) [inline Src/Newtonsoft.Json/Linq/JToken.cs]: Change this to "an instance of"

File: Src/Newtonsoft.Json/Linq/JToken.cs [hunk 3]
```diff
@@ -1925,7 +1925,7 @@ public T ToObject<T>()
...
```
Comment (JamesNK) [inline Src/Newtonsoft.Json/Linq/JToken.cs]: instance of

File: Src/Newtonsoft.Json/Linq/JToken.cs [hunk 4]
```diff
@@ -2048,7 +2048,7 @@ public object ToObject(Type objectType)
         }
 
         /// <summary>
-        /// Creates the specified .NET type from the <see cref="JToken"/> using the specified <see cref="JsonSerializer"/>.
+        /// Creates an object of the specified .NET type from the <see cref="JToken"/> using the specified <see cref="JsonSerializer"/>.
```
Comment (JamesNK) [inline Src/Newtonsoft.Json/Linq/JToken.cs]: instance of

File: Src/Newtonsoft.Json/Linq/JToken.cs [hunk 5]
```diff
@@ -2059,7 +2059,7 @@ public T ToObject<T>(JsonSerializer jsonSerializer)
...
```
Comment (JamesNK) [inline Src/Newtonsoft.Json/Linq/JToken.cs]: instance of

**Comment by stakx** [comment]
Done.

**Comment by JamesNK** [comment]
Perfect! Merged

**Pull request merged**